kanopi/saplings-media
# Add more options for image sizes

## Description
Teamwork Ticket(s): https://kanopi.teamwork.com/app/tasks/29470193
Start easy responsive image at 350px and increment by 350px up to 2000px.
All easy responsive images should have webp support.

## Steps to Validate
Install the sapling-media recipe with this PR and run a lighthouse test. We should not be getting dinged for image sizes.
If we are make note of the aspect ratio used and URL to help troubleshoot.

## Affected URL
[link_to_relevant_multidev_or_test_site](insert_link_here)

## Deploy Notes
Install and clear the cache a few times.
